docs/layout-of-source-files.rst

```diff
@@ -216,101 +216,19 @@ the code below creates new global symbols ``alias`` and ``symbol2`` which refere
 
   import {symbol1 as alias, symbol2} from "filename";
 
-Paths
------
+.. index:: virtual filesystem, source unit name, import; path, filesystem path
 
-In the above, ``filename`` is always treated as a path with ``/`` as directory separator,
-and ``.`` as the current and ``..`` as the parent directory.  When ``.`` or ``..`` is followed by a character except ``/``,
-it is not considered as the current or the parent directory.
-All path names are treated as absolute paths unless they start with the current ``.`` or the parent directory ``..``.
+Import paths
+------------
 
-To import a file ``filename`` from the same directory as the current file, use ``import "./filename" as symbolName;``.
-If you use ``import "filename" as symbolName;`` instead, a different file could be referenced
-(in a global "include directory").
+The paths used in imports in a general case do not have to be filesystem paths.
```

Comment:
Maybe we could also define that an import path is the parameter used within an import statement, `import "<import-path>"`.

Reply:
I added a sentence that says what an import path is in the "Syntax and Semantics" section above.

I have also rewritten the "Import paths" section here to say more about relation between an import path and a source unit name.